2/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:centos7
+FROM openshift/origin:latest 
```

Comment:
also @smarterclayton do you have any concerns w/ our jenkins image being based on openshift/origin?  (the benefit is it implicitly includes the latest oc client so we don't have to explicitly install it in our image and keep bumping the version)